docs/README.md

```diff
@@ -1,14 +1,20 @@
-# VTEX Country Flags
+📢 Use this project, [contribute](https://github.com/{OrganizationName}/{AppName}) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
 
-A React component that renders country flags as SVG icons.
+# VTEX COUNTRY FLAGS
 
-## API
+⚠️ **This is an ongoing, unsupported, unfinished and undocumented project. We do not guarantee any results after installation.** ⚠️
 
-### CountryFlag
+<!-- DOCS-IGNORE:start -->
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+<!-- DOCS-IGNORE:end -->
 
-This component receives the country code and renders that respective country's flag.
 
-#### Props
+This component receives the country code and renders that respective country's flag in SVG.
```

Comment:
```suggestion
The Country Flags app provides flags in SVG icons to be displayed according to the countries' ISO code.
```

Reply:
It would be nice if we could add a media (image/gif) showing the app once rendered!